common/placer1.cc

```diff
@@ -350,6 +351,8 @@ class SAPlacer
                     log_error("failed to place cell '%s' of type '%s'\n", cell->name.c_str(ctx), cell->type.c_str(ctx));
                 --iters;
                 ctx->unbindBel(ripup_target->bel);
+                if (ripup_target->type == id_SB_GB)
```

Comment:
This will break the ECP5 build, where SB_GB doesn't exist (this seems just to be debugging anyway)

Reply:
Doh .. yeah, not sure how I missed that :/